API-proposed.yaml

```diff
@@ -3299,12 +3632,64 @@ components:
           format: int64
         style:
           $ref: "#/components/schemas/OutputElementStyle"
-        xValues:
+        xValuesDescription:
           type: array
-          description: Series' X values
+          description: Series' X axis description
           items:
-            type: integer
-            format: int64
+            required:
+            - dataType
+            - label
+            - unit
+            type: object
+            properties:
+              dataType:
+                type: string
+                description: The type of data this axis represents
+                enum:
+                - NUMBER
+                - BINARY_NUMBER
+                - TIMESTAMP
+                - DURATION
+                - STRING
+                - TIME_RANGE
+              unit:
+                type: string
+                description: "Unit associated with this axis (e.g., ns, ms)"
+              label:
+                type: string
+                description: Label for the axis
+              axisDomain:
+                $ref: "#/components/schemas/AxisDomain"
+        yValuesDescription:
```

Comment:
There should be a ref to XYAxisDescription instead of listing the details here. The swagger document needs to be updated before regenerating this.

Looking at the swagger documentation, it needs to be change to:
@Schema(description = "Series' Y axis description", requiredMode = RequiredMode.REQUIRED)